strawberryfields/circuitspecs/gbs.py

```diff
@@ -45,13 +45,19 @@ class GBSSpecs(GaussianSpecs):
         "ThermalLossChannel",
         # single mode gates
         "Dgate",
-        "Xgate",
-        "Zgate",
+        #"Xgate",
+        #"Zgate",
         "Sgate",
         "Rgate",
         "Fouriergate",
         "BSgate"
     }
+    decompositions = {
+        "Xgate": {},
+        "Zgate": {},
+        "Fouriergate": {},
+        "S2gate":{}
```

Comment:
Still find this very very strange 🤔 This looks like an old test (below), weird that it passes in master without the S2gate defined here.

```python
"/home/travis/build/XanaduAI/strawberryfields/tests/frontend/test_program.py", line 700, in test_GBS_measure_fock_register_order
    prog = prog.compile("gbs")
  File "/home/travis/build/XanaduAI/strawberryfields/strawberryfields/program.py", line 511, in compile
    seq = db.decompose(self.circuit)
  File "/home/travis/build/XanaduAI/strawberryfields/strawberryfields/circuitspecs/circuit_specs.py", line 278, in decompose
    raise pu.CircuitError("The operation {} cannot be used with the target '{}'.".format(cmd.op.__class__.__name__, self.short_name))
strawberryfields.program_utils.CircuitError: The operation S2gate cannot be used with the target 'gbs'.
```